src/Tokenizer/Analyzer/ArgumentsAnalyzer.php

```diff
@@ -83,4 +83,50 @@ public function getArguments(Tokens $tokens, $openParenthesis, $closeParenthesis
 
         return $arguments;
     }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $argumentStart
+     * @param int    $argumentEnd
+     *
+     * @return array
```

Comment:
this array looks complicated. I must admit that looking on implementation I have no idea what will be the output - like is `name` obligatory, or could be empty. when i will get -1 instead of proper index.
probably new DTO object would be good.